sdk/ml/azure-ai-ml/tests/workspace/unittests/test_workspace_operations.py

```diff
@@ -75,6 +81,16 @@ def test_begin_create(
         mocker.patch("azure.ai.ml._arm_deployments.ArmDeploymentExecutor.deploy_resource", return_value=LROPoller)
         mock_workspace_operation.begin_create(workspace=Workspace(name="name"))
 
+    def test_begin_create_featurestore_kind(
```

Comment:
should maybe also add a e2e test for featurestore workspace kind

Reply:
will add in the next PR which is in top of it to add operations with feature store